src/backend/tests/work-packages-create.test.ts

```diff
@@ -13,14 +13,20 @@ describe('work package create', () => {
     };
 
     describe('validate inputs', () => {
+      const mockWbsNum = {
+        carNumber: 1,
+        projectNumber: 1,
+        workPackageNumber: 0
+      };
+
       const goodBody = {
         userId: 1,
         name: 'name',
         crId: 2,
-        projectId: 1,
+        projectWbsNum: mockWbsNum,
         startDate: '2015-10-06',
         duration: 1,
-        wbsElementIds: [1],
+        dependencies: [mockWbsNum],
```

Comment:
New separate ticket cuz this reminded me of additional edge case business logic to disallow https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/issues/535